cmd/ocsp-updater/main.go

```diff
@@ -624,14 +624,9 @@ func (updater *OCSPUpdater) missingReceiptsTick(ctx context.Context, batchSize i
 		// If the feature flag is enabled, only send the certificate to the missing
```

Comment:
I think this comment needs to be updated now that there isn't a feature flag being checked.